.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
+language: python
+python:
+  - "3.5"
+
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
It looks like you currently need sudo support in order to be able to run trusty. And it looks like above we need at least trusty.